### PR TITLE
chore: release main

### PR DESCRIPTION
**Pull request by origamiserviceuser**
:robot: I have created a release \*beep\* \*boop\*
---
<details><summary>@financial-times/o-layout: 5.0.6</summary>

### Bug Fixes

* add custom navigation to this.navAnchors in order to stop errors being thrown ([f49e9ea](https://www.github.com/Financial-Times/origami/commit/f49e9ea9cdb95fbe24d3cddef9d8e3481bf2b5e3))
* fallback to 16px if an o-layout__main is not on the page ([e737352](https://www.github.com/Financial-Times/origami/commit/e737352a73aefa9dcbc3cf7240b48a8bc5a1ecfd))
* only setup scrolling based heading highlighting if intersectionobserver exists ([8c5f869](https://www.github.com/Financial-Times/origami/commit/8c5f86971eb80564284ecaf41ae2fb4f8b27c69c))

### Performance Improvements

* improve performance and logic around how to highlight headings in the sidebar ([b995ae0](https://www.github.com/Financial-Times/origami/commit/b995ae0600c72fb052666ca051082a8d36ad700d))
</details>

This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -1,54 +1 @@
...
...oter":"9.0.2","components/o-footer-services":"4.0.3","components/o-forms":"9.0.2","components/o-ft-affiliate-ribbon":"5.0.1","components/o-grid":"6.0.1","components/o-header":"9.0.2","components/o-header-services":"5.0.2","components/o-icons":"7.0.2","components/o-labels":"6.0.1","components/o-layout":"5.0.6","components/o-lazy-load":"3.0.1","components/o-loading":"5.0.1","components/o-message":"5.0.1","components/o-meter":"3.0.1","components/o-normalise":"3.0.2","components/o-overlay":"4.0.1","components/o-quote":"5.0.2","components/o-share":"8.0.1","components/o-spacing":"3.0.2","components/o-stepped-progress":"3.0.1","components/o-subs-card":"6.0.1","components/o-syntax-highlight":"4.0.1","components/o-table":"9.0.3","components/o-tabs":"6.0.1","components/o-teaser":"6.0.3","components/o-teaser-collection":"4.0.1","components/o-toggle":"3.0.1","components/o-tooltip":"5.0.2","components/o-topper":"5.0.2","components/o-typography":"7.0.3","components/o-video":"7.0.3","components/o-viewport":"5.0.1","components/o-visual-effects":"4.0.2","libraries/o-autoinit":"3.0.1","libraries/o-brand":"4.0.3","libraries/o-errors":"5.1.1","libraries/o-tracking":"4.1.3","libraries/o-utils":"2.0.1"}
```
Comment (chee) [inline .release-please-manifest.json:1]: weird! it wants it to be one line? guess we should update the regenerate config script

File: components/o-layout/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,19 @@
...
+### Bug Fixes
+
+* add custom navigation to this.navAnchors in order to stop errors being thrown ([f49e9ea](https://www.github.com/Financial-Times/origami/commit/f49e9ea9cdb95fbe24d3cddef9d8e3481bf2b5e3))
+* fallback to 16px if an o-layout__main is not on the page ([e737352](https://www.github.com/Financial-Times/origami/commit/e737352a73aefa9dcbc3cf7240b48a8bc5a1ecfd))
+* only setup scrolling based heading highlighting if intersectionobserver exists ([8c5f869](https://www.github.com/Financial-Times/origami/commit/8c5f86971eb80564284ecaf41ae2fb4f8b27c69c))
+
+
+### Performance Improvements
+
+* improve performance and logic around how to highlight headings in the sidebar ([b995ae0](https://www.github.com/Financial-Times/origami/commit/b995ae0600c72fb052666ca051082a8d36ad700d))
```
Comment (chee) [inline components/o-layout/CHANGELOG.md:15]: :)

File: components/o-layout/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@financial-times/o-layout",
-  "version": "5.0.5",
+  "version": "5.0.6",
```
Comment (chee) [inline components/o-layout/package.json:3]: wow, as a patch :o

**Review by chee — APPROVED** [review]
BIG

**Review by chee — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:robot: Release for @financial-times/o-layout is at https://github.com/Financial-Times/origami/releases/tag/o-layout-v5.0.6 :sunflower:

**Pull request merged**